5.application_scopes.md

```diff
@@ -65,28 +65,22 @@ The spec defines the constituent parts of a scope.
 
 | Attribute | Type | Required | Default Value | Description |
 |-----------|------|----------|---------------|-------------|
-| `type` | `string` | Y | | Determines type of the application scope. The core types are defined [above](#core-application-scope-types) |
 | `allowComponentOverlap` | `bool` | Y | | Determines whether a component is allowed to be in multiple instances of this scope type simultaneously. When false, the runtime implementation MUST produce an error and stop deployment if an attempt is made to place a component into more than one instance of this scope type simultaneously. |
-| `parameters` | [`[]Parameter`](#parameter) | N | | The scope's configuration options. |
+| `definitionRef` | [`DefinitionRef`](#definitionRef) | Y | | Index of the scope definition. |
 
-The `type` field defines the type name for this type. Examples may be found in the [Core Application Scope Types](#core-application-scope-types) above. Extended scope types MUST NOT be in the `core.oam.dev` namespace.  However, they must follow the [group, version, kind notation](2.overview_and_terminology.md). For example: `scopes.example.com/v1.ExtendedNetworkScope`
-
-#### Parameter
-
-The parameters that a scope exposes to operators.
+### DefinitionRef
 
 | Attribute | Type | Required | Default Value | Description |
 |-----------|------|----------|---------------|-------------|
-| `name` | `string` | Y | | The parameter's name. Must be unique per scope. |
-| `description` | `string` | N | | A description of the parameter. |
-| `type` | `string` | Y | | The parameter's type (One of `string`, `boolean`, `number`). |
-| `required` | `boolean` | N |`false` | Whether a value _must_ be provided for the parameter. |
-| `default` | type indicated by `type` field | N | | The parameter's default value. |
+| `name` | `string` | Y | | The name of this scope definition. |
+| `version` | `string` | Y | | The version of this scope definition. |
+| `schematic` | `string` | N | | The schematic of this scope properties. The value is a [JSON schema](https://json-schema.org/). |
+
+A scope definitionRef __MUST__ must be unique by `name` and `version`. The `schematic` field is optional as the definitionRef could be simply an indexing for a schema. In this case a runtime itself __SHOULD__ store definitions and a definition __MUST__ contain a schematic which can validate scope spec.
 
-The `name` field allows Unicode letters, numbers and `_` and `-`. The `description` field allows those characters plus whitespace and punctuation characters.
 
 ## Deployment
-Application scope instances are defined and deployed in an application configuration. See [Application Configuration](7.application_configuration.md) for more information on deploying scopes.
+Application scope instances are defined and deployed by their standalone way.
 
 ## Core scope type definitions
```

Comment:
Can we also move all the definitions of core scopes to a separate place?

Reply:
removed